pkg/apis/serving/v1alpha1/kfservice_types.go

```diff
@@ -61,6 +61,8 @@ type TensorflowSpec struct {
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
 	// Defaults to requests and limits of 1CPU, 2Gb MEM.
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// Mount a PersistentVolumeClaim for model
+	PersistentVolumeClaim *PersistentVolumeClaim `json:"persistentVolumeClaim,omitempty"`
```

Comment:
how about we don't change TensorflowSpec/XGBoostSpec/SKLearnSpec. 
if we want to mount PVC, just add related information in kfservice metadata.annotations, then parse  deployment  metadata.annotations in new webhook.(now kfservice metadata.annotations will be merged into knative configuration automatically, and I think revision deployment merges it into its annotations in knative side automatically, too)

Reply:
Yes, that may be a better solution. Take That. Thanks HouGuang.